data/json/recipes/recipe_medsandchemicals.json

```diff
@@ -88,10 +88,8 @@
   "difficulty": 2,
   "time": 10000,
   "book_learn": [["adv_chemistry", 1] , ["textbook_chemistry", 1]],
-  "tools": [
-    [
-      [ "chemistry_set", 5 ]
-    ]
+  "qualities": [
+    {"id":"CHEM","level":1}
```

Comment:
back in the day we actually had acid rain that was strong enough to burn your skin (in the game).  I just assumed that the acid water in this recipe was an extension of that (acid from polution), just boiling it down to concentrate it.

But if I google "how do I make acid from salt" almost all the results are for hydrochloric acid so I think that's a valid premise